emoji-selector@maestroschan.fr/metadata.json

```diff
@@ -2,9 +2,18 @@
 	"description": "This extension provides a parametrable popup menu displaying most emojis, clicking on an emoji copies it to the clipboard. An appropriate font like 'Twitter Color Emoji' or 'JoyPixels Color' should be installed on your system for a better visual result.",
 	"name": "Emoji Selector",
 	"shell-version": [
+		"3.26",
+		"3.28",
+		"3.30",
+		"3.32",
 		"3.34",
 		"3.36",
-		"3.38"
+		"3.38",
+		"3.49",
+		"40.0",
+		"41.0",
+		"41.1",
+		"41.2"
```

Comment:
Just fyi, according to the docs (https://gjs.guide/extensions/overview/anatomy.html#shell-version) these should just be 

```
  "40",
  "41"
]
```

I realize we're waiting for the much larger PR to land but just wanted to mention this for all the people who aren't waiting and merging things in manually.